keyborg/src/FocusEvent.ts

```diff
@@ -80,93 +56,108 @@ export interface KeyborgFocusInEvent extends Event {
     details: KeyborgFocusInEventDetails;
 }
 
-export function callOriginalFocusOnly(element: HTMLElement): void {
-    const focus = element.focus as CustomFocusFunctionWithOriginal;
+/**
+ * Guarantees that the native `focus` will be used
+ */
+export function nativeFocus(element: HTMLElement): void {
+    const focus = element.focus as KeyborgFocus;
 
-    if (focus.__keyborgFocus) {
-        focus.__keyborgFocus.call(element);
+    if (focus.__keyborgNativeFocus) {
+        focus.__keyborgNativeFocus.call(element);
     } else {
         element.focus();
     }
 }
 
+/**
+ * Overrides the native `focus` and setups the keyborg focus event
+ */
 export function setupFocusEvent(win: Window): void {
-    const wnd = win as WindowWithHTMLElementAndLastFocusedElement;
+    const kwin = win as WindowWithKeyborgFocusEvent;
 
     if (!_canOverrideNativeFocus) {
-        _canOverrideNativeFocus = canOverrideNativeFocus(wnd);
+        _canOverrideNativeFocus = canOverrideNativeFocus(kwin);
     }
 
-    const origFocus = wnd.HTMLElement.prototype.focus;
+    const origFocus = kwin.HTMLElement.prototype.focus;
 
-    if ((origFocus as CustomFocusFunctionWithOriginal).__keyborgFocus) {
+    if ((origFocus as KeyborgFocus).__keyborgNativeFocus) {
         // Already set up.
         return;
     }
 
-    wnd.HTMLElement.prototype.focus = focus;
+    kwin.HTMLElement.prototype.focus = focus;
 
-    const data: KeyborgFocusEventData = wnd.__keyborgData = {
+    const data: KeyborgFocusEventData = kwin.__keyborgData = {
         focusInHandler: (e: FocusEvent) => {
             const target = e.target as HTMLElement;
+            if (!target) {
+                return;
```

Comment:
By the way, from the minification point of view `if(condition){ actions }` is a few bytes shorter than `if(!condition){return} actions` :)